.yarn/versions/19173632.yml

```diff
@@ -0,0 +1,10 @@
+releases:
+  fast-check: patch
+
+declined:
+  - "@fast-check/monorepo"
+
+undecided:
+  - "@fast-check/ava"
+  - "@fast-check/jest"
```

Comment:
Those two should be marked for patches too

Reply:
Done.